src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/GetCaseResponse.java

```diff
@@ -62,9 +62,6 @@ public class GetCaseResponse {
     @JsonProperty("stages")
     private List<SimpleStageDto> stages;
 
-    @JsonProperty("completed")
```

Comment:
We should maintain the same external API so there needs to be a getter to replace this.
```
@JsonProperty("completed")
public Boolean getCompleted() {
    return dateCompleted != null;
}
```
This should be included in tests as well.